### DISCUSSION TITLE
Re-export SEPA XML orders

### DISCUSSION THREAD

**Discussion by Nebelfuchs** [pretix/pretix#3411] [Q&A] [answered]
Hi there,

is there a way to re-export SEPA XML orders? We are having an issue with 4 orders in 1 xml and would like to re-export them one by one so we have 4 xml files to upload instead of one. Is there a way or do we have to talk to the bank?

**Comment by raphaelm** [comment, marked as answer]
There's no way in pretix, but you could use a text editor and change the XML files manually

**Reply by Nebelfuchs** [reply]
Thanks - will try to do that :)
